src/index.ts

```diff
@@ -1,10 +1,17 @@
-import { Users } from './services/users';
+import { HTTP } from './services/http';
 import { Logger } from './services/logger';
-
+import { Users } from './services/users';
 import type { User, ApiConfig } from './types';
+import IoCContainer from 'ioc-lite';
+
+const ioc = new IoCContainer();
```

Comment:
Why do you create an ioc container here if you already have one in src/ioc/index.ts ?

Reply:
Oh, yes, good point! Thank you.